src/changes_notes.txt

```diff
@@ -0,0 +1,137 @@
+
+Content of this pull request :
+------------------------------
```

Comment:
While I definitely appreciate the great description outlined in the PR, there's no need to add it as a separate file in the PR itself. :sweat_smile:

Reply:
I agree, I remove this file